solidity/dashboard/.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require("@keep-network/prettier-config-keep"),
+}
```

Comment:
We've got a `Dockerfile` that I believe can get formatted by `"prettier-plugin-sh"` plugin.

```suggestion
  plugins: ["prettier-plugin-sh"],
}
```